projects/uswds-components/src/lib/range-slider/range-slider.component.html

```diff
@@ -0,0 +1,15 @@
+<label class="usa-label" for="usa-range">Range slider</label>
```

Comment:
Let's remove the label here and let the user define one if needed. Easier that way than to have every form control piece in the library have some custom html or string text as input for label.